src/main/java/com/streamr/client/dataunion/JavaWrapperUtil.java

```diff
@@ -0,0 +1,93 @@
+package com.streamr.client.dataunion;
+
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.web3j.codegen.SolidityFunctionWrapperGenerator;
+
+import java.io.*;
+
+public class JavaWrapperUtil {
```

Comment:
Should we remove this class from this repository and add to Streamr internal utilities?

Reply:
It's not part of DU2 API. Make it package local. I think this should be part of the build process.

Reply:
contract changes very infrequently. I think this is fine. @hpihkala ?

Reply:
made package local